sonar-scala-plugin/build.gradle

```
@@ -1,7 +1,7 @@
 plugins {
     id "scala"
     id "org.scoverage" version "8.0.3"
-    id 'com.github.johnrengelman.shadow' version '6.1.0'
+    id 'com.github.johnrengelman.shadow' version '7.1.0'
```

How about going to the  end of the 7.1.X branch with [7.1.2]?(https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow/7.1.2)
```suggestion
    id 'com.github.johnrengelman.shadow' version '7.1.2'
```